src/ra_server_proc.erl

```diff
@@ -874,10 +874,16 @@ receive_snapshot(EventType, Msg, State0) ->
         {receive_snapshot, State1, Effects} ->
             {#state{conf = Conf} = State, Actions} =
                 ?HANDLE_EFFECTS(Effects, EventType, State1),
-            #conf{receive_snapshot_timeout = ReceiveSnapshotTimeout} = Conf,
-            {keep_state, State,
-             [{state_timeout, ReceiveSnapshotTimeout,
-               receive_snapshot_timeout} | Actions]};
+            TimeoutActions = case Msg of
```

Comment:
This is fine but can you format it like the rest of the code in the module. `case` and `end` should line up.

Reply:
Fixed up in 7851f9182ade17f9fd25da1b6e7ca52a6536f502.